Doc/library/contextlib.rst

```diff
@@ -267,7 +267,7 @@ Functions and classes provided:
 .. function:: suppress(*exceptions)
 
    Return a context manager that suppresses any of the specified exceptions
-   if they occur in the body of a with statement and then resumes execution
+   if they are raised in the body of a with statement and then resumes execution
```

Comment:
```suggestion
   if they are raised in the body of a :keyword:`with` statement and then resumes execution
```

Reply:
While we're here, let's make it a hyperlink like it should be. Could you also wrap this line so it doesn't exceed 80 chars?

Reply:
Yes, definitely. It's done.